### ISSUE TITLE
Support for Django 4.1

### ISSUE THREAD

**Issue by oscarmcm** [django-wiki/django-nyt#112]
Hi, is there a reason on why the Django version is locked to `<4.1`? I just checked the tests and as well the tests app and seems that everything works fine.

```shell
❯ python -V
Python 3.10.8

❯ pip freeze
asgiref==3.5.2
Django==4.1.3
-e git+ssh://git@github.com/oscarmcm/django-nyt.git@28a5afbc10b2a31c6be8a303524905f19077fdf2#egg=django_nyt
sqlparse==0.4.3

❯ ./runtests.py
Found 4 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
Starting django_nyt e-mail dispatcher
Sending to: bob@example.com
Sending to: alice@example.com
Starting django_nyt e-mail dispatcher
Starting django_nyt e-mail dispatcher
Daemon mode enabled, forking
PID: 98595
Starting send loop at 2022-11-10 14:25:47.268106
....
----------------------------------------------------------------------
Ran 4 tests in 1.226s

OK
Destroying test database for alias 'default'...
```

I'm happy to provide a patch if there's any in order to add support for `Django 4.1`, those are the changes I added:

```diff
diff --git a/runtests.py b/runtests.py
index e265075..47d70d6 100755
--- a/runtests.py
+++ b/runtests.py
@@ -57,6 +57,7 @@ settings.configure(
         'django.contrib.auth.middleware.AuthenticationMiddleware',
         'django.contrib.messages.middleware.MessageMiddleware',
     ],
+    DEFAULT_AUTO_FIELD='django.db.models.AutoField',
 )
 
 
diff --git a/setup.py b/setup.py
index 0a410be..593c709 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ setup(
     keywords=["django", "notification" "alerts"],
     packages=find_packages(),
     zip_safe=False,
-    install_requires=["django>=2.2,<4.1"],
+    install_requires=["django>=2.2,<4.2"],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
diff --git a/test-project/testproject/settings/__init__.py b/test-project/testproject/settings/__init__.py
index d33f83e..9d45293 100644
--- a/test-project/testproject/settings/__init__.py
+++ b/test-project/testproject/settings/__init__.py
@@ -37,8 +37,7 @@ MEDIA_URL = '/media/'
 
 STATIC_ROOT = os_path.join(PROJECT_PATH, 'static')
 STATIC_URL = '/static/'
-STATICFILES_DIRS = (
-)
+STATICFILES_DIRS = ()
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
@@ -154,3 +153,4 @@ if _enable_channels:
 
 
 NYT_ENABLE_ADMIN = True
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

**Comment by benjaoming** [comment]
This would be great to fix, can you do a PR, adding the support? It needs to go into `tox.ini` as well...

**Comment by oscarmcm** [comment]
@benjaoming I just created a PR that add supports to it #113

**Comment by benjaoming** [comment]
Released in 1.2.4 :+1:
